pallets/rmrk-market/src/weights.rs

```diff
@@ -0,0 +1,104 @@
+
+//! Autogenerated weights for `pallet_rmrk_market`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-11-13, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `Sergejs-MacBook-Air.local`, CPU: `<UNKNOWN>`
+//! EXECUTION: None, WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/rmrk-substrate
+// benchmark
+// pallet
+// --pallet
+// pallet-rmrk-market
+// --extrinsic
+// *
+// --output
+// pallets/rmrk-market/src/weights.rs
+// --steps
+// 50
+// --repeat
+// 20
+// --chain
+// dev
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight}};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for pallet_rmrk_market.
+pub trait WeightInfo {
+	fn buy() -> Weight;
```

Comment:
When running the benchmark the generated output makes `WeightInfo` a struct. I also noticed when running the benchmark for the `rmrk-core` pallet that it also makes `WeightInfo` a struct. @Maar-io does the `WeightInfo` trait need to be defined by hand?

Reply:
You can do it manually or with template file. Please find mention of template in the cheetsheet above